ksqldb-rest-app/src/main/java/io/confluent/ksql/api/server/ServerVerticle.java

```diff
@@ -52,8 +54,11 @@
   // CHECKSTYLE_RULES.ON: ClassDataAbstractionCoupling
   private static final Logger log = LoggerFactory.getLogger(ServerVerticle.class);
 
-  private static final Set<String> NEW_API_ENDPOINTS = ImmutableSet
-      .of("/query-stream", "/inserts-stream", "/close-query");
+  private static final Set<String> NON_PROXIED_ENDPOINTS = ImmutableSet
+      .of("/query-stream", "/inserts-stream", "/close-query", "/ksql");
+
+  // Quick switch so we can easily revert to not serving ported endpoints directly
+  private static final boolean SERVE_PORTED_ENDPOINTS = true;
```

Comment:
Thanks for including this. Hopefully we won't need it but it's reassuring to know we can easily undo this change if something comes up.